src/Analysis/Ast/Test/GenericsTests.cs

```diff
@@ -1091,6 +1090,37 @@ class Simple(unittest.TestCase):
             analysis.Should().HaveVariable("x").Which.Should().HaveType("TypeError");
         }
 
+        [TestMethod, Priority(0)]
+        public async Task SpecializedGenericPartialParams() {
+            const string code = @"
+from typing import Dict, List, Tuple, Iterator, KeysView, ValuesView, ItemsViews, Sequence, Generic, TypeVar
```

Comment:
Does the test provided in the issue description pass?

Reply:
Yes it does.

Reply:
Your test is included in this one.